studio/lib/queries/navigation.ts

```diff
@@ -35,9 +35,9 @@ export const NAV_QUERY = groq`
         }
       }
     }
-  }[1]
+  }
```

Comment:
This will return a list of navigation managers, breaking code expecting it to be a single object.
```suggestion
  }[0]
```

Reply:
I'm double checking.

Navigation manager along with a few other documents cannot be [] due to a custom structure (DeskStructure) so that might be why I've changed it. I'll have to look through and remember what I did on Friday 😅

Reply:
Just me forgetting to push one commit. I've also added an extra check for proper render of different state changes in layout.